CMakeLists.txt

```
@@ -27,7 +27,8 @@ endfunction()
 # DEPRECATED! Should be merged into add_module_library.
 function(enable_module target)
   if (MSVC)
-    set(BMI ${CMAKE_CURRENT_BINARY_DIR}/${target}.ifc)
+    string(REPLACE "/" "\\" BIN_DIR ${CMAKE_CURRENT_BINARY_DIR})
+    set(BMI ${BIN_DIR}\\${target}.ifc)
```

You can reuse existing CMake functionality:
```suggestion
    file(TO_NATIVE_PATH "${CMAKE_CURRENT_BINARY_DIR}/${target}.ifc" BMI)
```